ftw/publisher/sender/browser/views.py

```diff
@@ -127,24 +128,33 @@ def __call__(self, no_response=False, msg=None, *args, **kwargs):
         # create Job
         portal = self.context.portal_url.getPortalObject()
         queue = IQueue(portal)
-        queue.createJob('move', self.context, username, )
+
+        additional_data = {'move_data': {
+            'newName': event.newName,
+            'newParent': get_site_relative_path(event.newParent),
+            'newTitle': event.object.Title().decode('utf-8'),
+            'oldName': event.oldName,
+            'oldParent': get_site_relative_path(event.oldParent),
+        }}
+
+        queue.createJob('move', self.context, username,
+                        additional_data=additional_data)
         self.logger.debug('Created "%s" Job for "%s" at %s' % (
-                'move',
-                self.context.Title(),
-                '/'.join(self.context.getPhysicalPath()),
-                ))
+            'move',
+            self.context.Title(),
+            '/'.join(self.context.getPhysicalPath()),
+        ))
         # status message
         if msg is None:
             msg = _(u'Object move/rename action has been added to the queue.')
-        IStatusMessage(self.request).addStatusMessage(
-            msg,
-            type='info'
+            IStatusMessage(self.request).addStatusMessage(
```

Comment:
Opps, that's an accident, thanks for pointing out!
(caused by emacs' auto-format)